tests/story-based-tests.snapshot-test.js

```diff
@@ -26,9 +26,13 @@ const port = process.env.PORT || 8002;
 app.use(
 	'/assets',
 	express.static(
-		`${rootPath}/node_modules/@salesforce-ux/design-system/assets/`
+		path.join(
```

Comment:
This line causes all CSS to fail, because the CSS is not loading now there are no styles present. Only 4 snapshots should be failing on an upgrade to 2.9.x.

I'm seeing this week that segmenting the tests might help a little with performance at around 1.5 minutes faster.